recipes/multiprocessing_on_dill/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "multiprocessing_on_dill" %}
+{% set version = "3.5.0a4" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d6d50c300ff4bd408bb71eb78725e60231039ee9b3d0d9bb7697b9d0e15045e7" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python 
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - dill
+
+test:
+  imports:
+    - multiprocessing_on_dill
+    - multiprocessing_on_dill.dummy
+
+about:
+  home: https://github.com/sixty-north/multiprocessing_on_dill
+  license: PSFL 
+  license_family: OTHER
+  license_file: LICENSE
```

Comment:
Please ask upstream to add a `MANIFEST.in` and include the license file. That way it will be included in future `sdist`s and we can just get it from there.

Reply:
I've asked upstream but got no response since Aug 13. [#1](https://github.com/sixty-north/multiprocessing_on_dill/issues/1) - so we might have to come up with a solution here.